src/lib.rs

```diff
@@ -329,6 +416,8 @@ struct TableInner {
 }
 
```

Comment:
Since we are already so thorough with documentation, which I love, and we've documented its methods, it'd be good to document the TableInner itself